pro/cypress/support/helpers.ts

```diff
@@ -3,6 +3,21 @@ import {
   MOCKED_BACK_ADDRESS_STREET,
 } from '../support/constants.ts'
 
+/**
+ * This function takes a string[][] as a DataTable representing the data
```

Comment:
je ne suis pas dev, mais ça me choque de dire qu'une chaîne de caractères (string) est un tableau à deux dimensions :thinking: 
```suggestion
 * This function takes a table[][] as a DataTable representing the data
```

Reply:
Non c'est bien le type que décrit Typescript

![CleanShot 2024-12-11 at 10 42 08](https://github.com/user-attachments/assets/a4640a48-7230-426e-bc89-3bb1cc3bad7d)